File: app/src/main/java/org/stepic/droid/ui/fragments/TextStepFragment.java
package org.stepic.droid.ui.fragments;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.squareup.otto.Subscribe;

import org.stepic.droid.R;
import org.stepic.droid.base.StepBaseFragment;
import org.stepic.droid.events.comments.NewCommentWasAddedOrUpdateEvent;
import org.stepic.droid.events.steps.StepWasUpdatedEvent;

import butterknife.ButterKnife;

public class TextStepFragment extends StepBaseFragment {
    @Nullable
    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        View v = inflater.inflate(R.layout.fragment_text_step, container, false);
        unbinder = ButterKnife.bind(this, v);
        return v;
    }

    @Subscribe
    public void onNewCommentWasAdded(NewCommentWasAddedOrUpdateEvent event) {
        super.onNewCommentWasAdded(event);

    }

    @Subscribe
    public void onStepWasUpdated(StepWasUpdatedEvent event) {
        super.onStepWasUpdated(event);
    }
}

File: app/src/main/java/org/stepic/droid/ui/fragments/TextStepFragment.kt
package org.stepic.droid.ui.fragments

import android.os.Bundle
import android.os.Handler
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import kotlinx.android.synthetic.main.fragment_text_step.*
import org.stepic.droid.R
import org.stepic.droid.base.StepBaseFragment

class TextStepFragment : StepBaseFragment() {
    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View
            = inflater.inflate(R.layout.fragment_text_step, container, false)

    override fun attachStepTextWrapper() {
        stepTextWrapper.attach(stepContainer)
    }

    override fun detachStepTextWrapper() {
        stepTextWrapper.detach(stepContainer)
    }
}
